.github/workflows/test.yaml

```
@@ -18,12 +19,17 @@ jobs:
         uses: ./
         with:
           version: latest
-          config: testdata/lava.yaml
+          config: ${{ env.WANT_CONFIG }}
         continue-on-error: true
       - name: Print status
         run: 'echo "Lava status: ${{ steps.lava.outputs.status }}"'
       - name: Print report
         run: 'cat "${{ steps.lava.outputs.report }}"'
+      - name: Report unexpected config file
+        if: ${{ !endsWith(steps.lava.outputs.config, env.WANT_CONFIG) }}
```

Can't you just use `!=`?

```suggestion
        if: ${{ steps.lava.outputs.config != env.WANT_CONFIG }}
```